app/scripts/models/reliers/oauth.js

```diff
@@ -97,21 +97,6 @@ define([
       return RelierKeys.deriveRelierKeys(keys, uid, this.get('clientId'));
     },
 
-    pickResumeTokenInfo: function () {
```

Comment:
I extracted this into a decorator so that it can be used by other models (in particular, the User model with the uniqueUserId, for a follow on PR)